irs-edc-client/src/main/java/org/eclipse/tractusx/irs/edc/client/policy/Constraint.java

```diff
@@ -39,13 +39,14 @@
 public class Constraint {
 
     @Schema(implementation = String.class, example = "string")
-    @JsonAlias({"odrl:leftOperand"})
+    @JsonAlias({ "odrl:leftOperand" })
```

Comment:
why were these spaces inserted?

Reply:
code formatter does this automatically. however the "{}" are superfluous here => removed completely